src/System.Net.Security/ref/project.json

```diff
@@ -9,9 +9,9 @@
     "System.Threading.Tasks": "4.0.0"
   },
   "frameworks": {
-    "netstandard1.3": {
+    "netstandard1.6": {
```

Comment:
Correct you cannot add more apis to netstandard16. Given this API doesn't exist anywhere else you actually should add it to only .NET Core similar to what @ericstj is doing in https://github.com/dotnet/corefx/pull/11272.